src/bdev.rs

```diff
@@ -57,6 +57,8 @@ where
     _data: PhantomData<BdevData>,
 }
 
+unsafe impl<T: BdevOps> Send for Bdev<T> {}
```

Comment:
Could we add a comment explaining why this is safe?

Reply:
When you have to manually mark a struct as Send, Rust requires it to be unsafe: https://doc.rust-lang.org/nomicon/send-and-sync.html